src/python_framework/nwm_network_commandline.py

```diff
@@ -0,0 +1,84 @@
+import nhd_network_utilities as nnu
+import recursive_print
+import time
+import os
+import argparse
+
+def _handle_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        "--debuglevel",
+        help="Set the debuglevel",
+        dest="debuglevel",
+        choices=["0", "1", "2", "3"],
+        default=0,
+    )
+    parser.add_argument(
+        "-w",
+        "--break_at_waterbodies",
+        help="Use the waterbodies in the route-link dataset to divide the computation (leave blank for no splitting)",
+        dest="break_network_at_waterbodies",
+        action="store_true",
+    )
```

Comment:
This is probably something we should eventually talk to Nels about and see if we can use this parser with some customization. (Some flags are not relevant, depending on the file). But this will work for now, with the noted comments fixed.